packages/typespec-test/test/widget_dpg/spec/main.tsp

```diff
@@ -176,8 +176,4 @@ interface Budgets {
     // the same name as Widgets.createOrReplace
     @route("/widgets/createOrReplace")
     op createOrReplace is ResourceOperations.LongRunningResourceCreateOrReplace<User>;
-
-    // the patch case for lro
-    @route("/widgets/createOrUpdate")
```

Comment:
Patch LRO in MPG is okay but there exists issue in getLroMetadata for data-plane tamplate. But we don't have a real case for PATCH LRO in data-plane yet.

So I tentatively remove this operation. Let's fix this when we have a real case.